style.css

```diff
@@ -1,113 +1,503 @@
+  line-height: 32px;
+  align-items: center;
+  color: #fff5e1;
+  margin-top: 0;
+}
+
+.littleParagraph {
+  font-style: normal;
+  font-size: 16px;
+  margin-top: -24px;
+  font-weight: 400;
+  line-height: 24px;
+  padding: 0 24px 0 24px;
+  font-family: 'Poppins', sans-serif;
+  color: #fff;
+}
+
+.connectIcons {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  gap: 24px;
+  list-style: none;
+  margin-left: 0;
+  margin-top: 0;
+  width: 200px;
+  height: 40px;
+}
+
+.background {
+  position: absolute;
+  bottom: 175px;
+  right: -18px;
+  width: 150px;
+  height: 100px;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+/* from here adding new things */
+.SecondPage {
+  margin-top: 520px;
+  justify-items: centre;
+  display: grid;
+  grid-template-columns: 100%;
+  width: 100%;
+  grid-template-rows: 250px repeat(3, auto);
+  min-height: 90px;
+  margin-left: -8px;
+}
+
+.MyRecentWorksTop {
+  display: flex;
+  font-size: 40px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #1c1a19;
+  margin-top: 142px;
+  width: 327px;
+  height: 35px;
+  margin-left: 27px;
+}
+
+.AboutLine1 {
+  width: 327px;
+  top: 814px;
+  position: absolute;
+  border: 1px solid #212121;
+  align-items: center;
+  margin-left: 26px;
+}
+
+.mainImage {
+  display: inline;
+  width: 327px;
+  height: 237px;
+  margin-left: 27px;
+  margin-top: 5px;
+}
+
+.multiPostStories {
+  display: flex;
+  align-items: center;
+  font-family: 'crete Round', sans-serif;
+  font-style: normal;
+  font-size: 32px;
+  line-height: 44px;
+  width: 327px;
+  height: 36px;
+  color: #091e42;
+  margin-top: 14px;
+  margin-left: 27px;
+  margin-bottom: 0;
+}
+
+.pageInformation {
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  width: 327px;
+  height: 147px;
+  line-height: 24px;
+  margin-top: 12px;
+  color: #091e42;
+  margin-left: 27px;
+}
+
+.techStacks {
+  display: flex;
+  gap: 12px;
+  list-style: none;
+  margin-left: -13px;
+}
+
+.stackButton {
+  font-weight: 400;
+  font-size: 15px;
+  text-align: centre;
+  text-decoration: none;
+  display: inline-block;
+  background-color: #fff;
+  color: #091e42;
+  border: 2px solid #8993a4;
+  padding: 10px 12px;
+  margin-top: 12px;
+}
+
+.firstProjectButton {
+  background-color: #ff6b00;
+  border: none;
+  color: #fff;
+  padding: 12px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  display: inline-block;
+  cursor: pointer;
+  width: 124px;
+  margin-top: 4px;
+  margin-left: 27px;
+}
+
+.firstProjectButton:hover {
+  border: #f55800;
+  color: #f55800;
+  background-color: #fff5e1;
+}
+
+.ThirdPage {
+  display: grid;
+  justify-items: center;
+  grid-template-rows: repeat(6, 370px);
+  grid-template-columns: 100px;
+  width: 100px;
+  margin-left: 138px;
+}
```

Comment:
- [ ] As required by project requirements i recommend you lay out you cards using flexbox

Reply:
- [ ] Please use Flexbox as requested by the previous reviewer. It would give you more flexibility in aligning your content and it is responsive 📱